source/_plugins/snippet.rb

```diff
@@ -30,9 +27,10 @@ def render(context)
       if @content
         snippet = @content
         if @id
-          snippet = snippet[/#{Regexp.escape("<!-- START SNIPPET: " + @id + " -->")}[^\n]*\n(.*?)\n[^\n]*#{Regexp.escape("<!-- END SNIPPET: " + @id + " -->")}/m, 1]
+          snippet = snippet[/#{Regexp.escape("START SNIPPET: " + @id)}[^\n]*\n(.*?)\n[^\n]*#{Regexp.escape("END SNIPPET: " + @id)}/m, 1]
```

Comment:
One more thing, right now the whole snippet will be included, also we comments:
```
<!-- START SNIPPET: method-example -->
<s:form action="baz">
    <s:textfield label="Enter your name" name="person.name"/>
    <s:submit value="Create person"/>
    <s:submit method="anotherMethod" value="Cancel"/>
</s:form>
<!-- END SNIPPET: method-example -->
```

http://struts.apache.org/core-developers/action-mapper.html

Reply:
Thank you! I examined and saw it's not a snippet tag there at all; It's hard-coded raw text block :P

Reply:
LoL! That's good :)